server/helpers/generate-jwt.ts

```diff
@@ -0,0 +1,22 @@
+import jwt from 'jsonwebtoken';
+
+const generateJWT = (uid: string) => {
+  return new Promise((resolve, reject) => {
+    const payload = {
+      uid
+    };
+
+    jwt.sign(payload, process.env.SECRETORPRIVATEKEY || '', {
+      expiresIn: '24h'
```

Comment:
24 hours might be too much, how about we lower this to 30m or something like that?